_data/sprint_teams.yml

```
@@ -0,0 +1,71 @@
+- name: Ada
+  namesake_markdown: "[Ada Lovelace](https://en.wikipedia.org/wiki/Ada_Lovelace), one of the world's first computer programmers"
+  focus_area: Unsupervised remote proofing
+  slack_channel_name: login-team-ada
+  slack_channel_url: https://gsa-tts.slack.com/archives/CNCGEHG1G
+  slack_group_names:
+  - login-ada
+
+- name: Agnes
+  namesake_markdown: "[Agnes Meyer Driscoll](https://www.nsa.gov/History/Cryptologic-History/Historical-Figures/Historical-Figures-View/Article/1623020/agnes-meyer-driscoll/), a navy cryptanalyst, known as the '[first lady of naval cryptology](https://www.hmdb.org/m.asp?m=106127)'"
+  focus_area: Attempts API
+  slack_channel_name: login-team-loggerhead
+  slack_channel_url: https://gsa-tts.slack.com/archives/C03N6KQBAKS
+
+- name: Grace
+  namesake_markdown: "[Admiral Grace Hopper](https://en.wikipedia.org/wiki/Grace_Hopper), a computer scientist and navy officer"
+  focus_area: KYC/Non-biometric verification
+  slack_channel_name: login-team-grace
+  slack_channel_url: https://gsa-tts.slack.com/archives/C0184P9SCJ0
+  slack_group_names:
+  - login-grace
+
+- name: IDVA
+  focus_area: "Equity Study & API"
+  slack_channel_name: identity-idva
+  slack_channel_url: https://gsa-tts.slack.com/archives/C017KUF5DL4
+
+- name: Joy
+  namesake_markdown: "[Dr. Joy Buolamwini](https://en.wikipedia.org/wiki/Joy_Buolamwini), a computer scientist, artist, and activist who has published foundational work on algorithmic discrimination."
+  focus_area: In Person Proofing
+  slack_channel_name: login-team-joy
+  slack_channel_url: https://gsa-tts.slack.com/archives/C03FA4VBN76
+  slack_group_names:
+  - login-joy-engineers
+  - login-joy-designers
```

Should we include the broader team handle as well?

```suggestion
  slack_group_names:
  - login-joy
  - login-joy-engineers
  - login-joy-designers
```

---

went with singular slack handle approach in b05fb04d707555f4f952ee548e91a1854bf022e1